product_pricelist_auto_mailing/models/product_pricelist_item.py

```diff
@@ -0,0 +1,13 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = 'product.pricelist.item'
+
+    def write(self, vals_list):
+        if self.env.user.id in self.env.company.user_notifying_price_changes_ids.ids:
+            self.pricelist_id.mailable_user_ids = [(4, self.env.user.id)]
```

Comment:
Not sure if this notation will always work. `write` is a multi method, so seems to be more accurate this, as `pricelist_id` could be one ore more pricelists (not tested), could you check?
```suggestion
            self.pricelist_id.write({"mailable_user_ids": [(4, self.env.user.id)]})
```